### ISSUE TITLE
Include command line client

### ISSUE THREAD

**Issue by nichtich** [maxlath/hub#7]
~~~
$ hub Q3
https://en.wikipedia.org/wiki/Universe
$ hub viaf:24597135
https://en.wikipedia.org/wiki/Isaac_Asimov
$ hub Q11 || echo "not found"
not found
$ hub -o Q184226 -s wikiquote -l fr 
   # opens https://fr.wikiquote.org/wiki/Gilles_Deleuze in browser
$ LANGUAGE=de_DE hub -j Q3
{
  "origin": {
    "id": "Q2"
  },
  "destination": {
    "langs": [
      "de"
    ],
    "sites": [
      "wiki"
    ],
    "preferedSitelink": {
      "site": "dewiki",
      "title": "Erde",
      "badges": [
        "Q17437798"
      ]
    },
    "url": "https://de.wikipedia.org/wiki/Erde"
  }
}
~~~

**Comment by maxlath** [comment]
I was thinking to something of this kind to integrate to `wikidata-cli`, replacing or extending [`wd open`](https://github.com/maxlath/wikidata-cli/blob/master/docs/read_operations.md#wd-open), would that work for you or do you think it should be a standalone command line tool?

**Comment by nichtich** [comment]
Extending `wd open` makes sense. By now `open` just creates an URL without checking (try `wd open Q11`) so better 

* pass the query to `open` through hub
* don't open or print URL if not found but exit with error code
* add option `-j, --json` to print JSON instead of opening URL

I can create `alias hub='wd open -u'` in my shell

**Comment by maxlath** [comment]
`wikidata-cli v6.4.0` ships with a `wd hub` command (as I found it easier than re-writting `wd open` while either deprecating or making breaking changes), could you give it a try?

**Comment by maxlath** [comment]
patch https://github.com/maxlath/wikidata-cli/commit/47d7bd1 published in `v6.4.1`
